src/Sarif.FunctionalTests/ConverterTestData/Pylint/PyLint-01.json.sarif

```diff
@@ -0,0 +1 @@
ssage":"Invalid constant name \"explained_variance\"","locations":[{"analysisTarget":{"uri":"OneDrive%20-%20Microsoft/ML/K-Means-Text/kmeans.py","region":{"startLine":128,"startColumn":4}}}]},{"ruleId":"(invalid-name)","level":"warning","message":"Invalid constant name \"km\"","locations":[{"analysisTarget":{"uri":"OneDrive%20-%20Microsoft/ML/K-Means-Text/kmeans.py","region":{"startLine":139,"startColumn":4}}}]},{"ruleId":"(invalid-name)","level":"warning","message":"Invalid constant name \"km\"","locations":[{"analysisTarget":{"uri":"OneDrive%20-%20Microsoft/ML/K-Means-Text/kmeans.py","region":{"startLine":142,"startColumn":4}}}]},{"ruleId":"(invalid-name)","level":"warning","message":"Invalid constant name \"t0\"","locations":[{"analysisTarget":{"uri":"OneDrive%20-%20Microsoft/ML/K-Means-Text/kmeans.py","region":{"startLine":146}}}]},{"ruleId":"(invalid-name)","level":"warning","message":"Invalid constant name \"original_space_centroids\"","locations":[{"analysisTarget":{"uri":"OneDrive%20-%20Microsoft/ML/K-Means-Text/kmeans.py","region":{"startLine":166,"startColumn":8}}}]},{"ruleId":"(invalid-name)","level":"warning","message":"Invalid constant name \"order_centroids\"","locations":[{"analysisTarget":{"uri":"OneDrive%20-%20Microsoft/ML/K-Means-Text/kmeans.py","region":{"startLine":167,"startColumn":8}}}]},{"ruleId":"(invalid-name)","level":"warning","message":"Invalid constant name \"order_centroids\"","locations":[{"analysisTarget":{"uri":"OneDrive%20-%20Microsoft/ML/K-Means-Text/kmeans.py","region":{"startLine":169,"startColumn":8}}}]},{"ruleId":"(invalid-name)","level":"warning","message":"Invalid constant name \"terms\"","locations":[{"analysisTarget":{"uri":"OneDrive%20-%20Microsoft/ML/K-Means-Text/kmeans.py","region":{"startLine":171,"startColumn":4}}}]},{"ruleId":"(wrong-import-order)","level":"warning","message":"standard import \"import logging\" should be placed before \"from sklearn.datasets import fetch_20newsgroups\"","locations":[{"analysisTarget":{"uri":"OneDrive%20-%20Microsoft/ML/K-Means-Text/kmeans.py","region":{"startLine":18}}}]},{"ruleId":"(wrong-import-order)","level":"warning","message":"standard import \"from optparse import OptionParser\" should be placed before \"from sklearn.datasets import fetch_20newsgroups\"","locations":[{"analysisTarget":{"uri":"OneDrive%20-%20Microsoft/ML/K-Means-Text/kmeans.py","region":{"startLine":19}}}]},{"ruleId":"(wrong-import-order)","level":"warning","message":"standard import \"import sys\" should be placed before \"from sklearn.datasets import fetch_20newsgroups\"","locations":[{"analysisTarget":{"uri":"OneDrive%20-%20Microsoft/ML/K-Means-Text/kmeans.py","region":{"startLine":20}}}]},{"ruleId":"(wrong-import-order)","level":"warning","message":"standard import \"from time import time\" should be placed before \"from sklearn.datasets import fetch_20newsgroups\"","locations":[{"analysisTarget":{"uri":"OneDrive%20-%20Microsoft/ML/K-Means-Text/kmeans.py","region":{"startLine":21}}}]}]}]}
```

Comment:
I am not sure why GitHub is showing json files as binary file.
pylint <python file> --output-format=json | out-file Pylint-01.json.